r/tests/testthat/test-atac_utils.R

```diff
@@ -154,6 +154,7 @@ test_that("write_insertion_bedgraph works", {
 })
 
 test_that("write_insertion_bed works", {
+  skip_on_os("windows") # TODO: threads >1 causes parallel::mclapply issues on windows
```

Comment:
As discussed, we should set Windows to use 1 thread as a fallback for now, and get a fix through moving the multithreading to C++